extensions/ql-vscode/src/variant-analysis/variant-analysis-view.ts

```diff
@@ -218,9 +218,14 @@ export class VariantAnalysisView
   }
 
   private getTitle(variantAnalysis: VariantAnalysis | undefined): string {
-    return variantAnalysis
-      ? `${variantAnalysis.query.name} - Variant Analysis Results`
-      : `Variant Analysis ${this.variantAnalysisId} - Results`;
+    if (variantAnalysis) {
+      if (variantAnalysis.queries) {
+        return `Variant Analysis using multiple queries - Results`;
+      }
+
+      return `${variantAnalysis.query.name} - Variant Analysis Results`;
+    }
+    return `Variant Analysis ${this.variantAnalysisId} - Results`;
```

Comment:
I find this set of if-statements a little hard to read. Possibly because all of the return statements are at different indentation levels and the whitespace/newline implies that the second two return statements are grouped together more than they should be.

Feel free to ignore this suggestion or do something else, but I suggest the following:
```suggestion
    if (!variantAnalysis) {
      return `Variant Analysis ${this.variantAnalysisId} - Results`;
    }

    if (variantAnalysis.queries) {
      return `Variant Analysis using multiple queries - Results`;
    } else {
      return `${variantAnalysis.query.name} - Variant Analysis Results`;
    }
```

Get the "no variant analysis" case out of the way first, and then give equal weight to the single-query and multi-query cases.